Misc/NEWS.d/next/Library/2018-05-15-15-03-48.bpo-28612.E9dz39.rst

```diff
@@ -0,0 +1,2 @@
+Added support for optional Site Map extension to urllib robotparser. Patch by
```

Comment:
It would be nice to add some links to the new `site_maps` (:meth:\`RobotFileParser.site_maps() <urllib.robotparser.RobotFileParser.site_maps>\` -- untested, you'll need to try it locally :)) method or to the `urllib.robotparser` (:mod:\`urllib.robotparser\`) module.